Use double quotes for log messages written to syslog

treydock opened the pull request:
**Description of the Pull Request (PR):**

This change will help tools like Splunk parse the key=value where single quotes are added to value but double quotes are not.

Example of new message output from running `make test`

```
Mar 27 09:09:04 build-el7 Singularity: action-suid (U=20821,P=32716)> USER="tdockendorf", IMAGE="container.img", COMMAND="exec"
```

**Checkoff for all PRs:**

- [x] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [x] I have added changes to the [CHANGELOG](https://github.com/singularityware/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [singularityware](https://www.github.com/singularityware/singularityware.github.io) documentation base.
- [x] I have tested this PR locally with a `make test`
- [x] This PR is NOT against the project's master branch
- [ ] I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- [x] This PR is ready for review and/or merge


Attn: @singularityware-admin

GodloveD commented on src/action.c:
```diff
@@ -145,7 +145,7 @@ int main(int argc, char **argv) {
     envar_set("SINGULARITY_APPNAME", singularity_registry_get("APPNAME"), 1);
 
-    singularity_message(LOG, "USER=%s, IMAGE='%s', COMMAND='%s'\n", singularity_priv_getuser(), singularity_image_name(&image), singularity_registry_get("COMMAND"));
+    singularity_message(LOG, "USER=%s, IMAGE=\"%s\", COMMAND=\"%s\"\n", singularity_priv_getuser(), singularity_image_name(&image), singularity_registry_get("COMMAND"));
```
Is there any reason not to wrap the USER string as well?

treydock replied:
For at least Splunk the double quotes are only necessary when the value in key=value contains a space.  It's possible a space may exist, I would assume, for IMAGE, but probably not for USER.

GodloveD reviewed (changes requested):
This seems like a sensible change to me.  I'm going to ask for just a few more changes.
1. Can you also wrap the user string in ""?
2. Can you make an entry in the CHANGELOG.md?
3. Can you rebase this against development-2.x?
Thanks for taking the time to identify this need and provide a fix!

treydock committed:
Use double quotes for log messages written to syslog

This change will help tools like Splunk parse the key=value where double quotes are added to value.
